packages/open-api-gateway/src/project/samples/typescript.ts

```diff
@@ -30,6 +30,10 @@ export interface TypescriptSampleCodeOptions {
    * Whether or not to generate sample code
    */
   readonly sampleCode?: boolean;
+  /**
+   * Source code directory
+   */
+  readonly srcdir: string;
```

Comment:
Shouldn't this be optional? We should also update the jsdocs with `@default`.

Reply:
This is internal - a srcdir is needed for generating the test code and imo it's better to enforce it's passed in because it needs to be the srcdir for the project that's including the sample code :). It's used in the test code as:

```ts
import { Api } from "../${options.srcdir}";
```